packages/cli/lib/axe-test-urls.js

```diff
@@ -1,7 +1,7 @@
 'use strict';
 
 const WebDriver = require('selenium-webdriver');
-const AxeBuilder = require('@axe-core/webdriverjs');
+const AxeBuilder = require('@axe-core/webdriverjs').default;
```

Comment:
This indicates a breaking change. We can't ship this until we're ready for a SEMVER major release.

If we want this to land sooner than that, we'll want to do some hackery the TS:

```
export default AxeBuilder
```

should be:

```
export = AxeBuilder
```

Alternatly, we could add a `src/index.ts` which looks like:

```
import AxeBuilder from './AxeBuilder'

// Enable ESM/TS imports while we wait for a SEMVER major release.
AxeBuilder.default = AxeBuilder

export = AxeBuilder
```